zenoh/src/lib.rs

```diff
@@ -356,10 +360,10 @@ where
 pub fn init(runtime: Runtime) -> InitBuilder {
     InitBuilder {
         runtime,
-        publications_destination: Locality::Any,
-        subscribers_origin: Locality::Any,
-        queries_destination: Locality::Any,
-        queryables_origin: Locality::Any,
+        publications_destination: Locality::default(),
```

Comment:
`Locality` should be only on subscriber and queryable side for the time being at least.